Libplanet/Net/Swarm.cs

```diff
@@ -1560,18 +1560,7 @@ CancellationToken cancellationToken
                         );
                         break;
                     }
-
-                    // We can't recover with OperationCanceledException and
-                    // ObjectDisposedException. so just re-throw them.
-                    catch (ObjectDisposedException)
-                    {
-                        throw;
-                    }
-                    catch (OperationCanceledException)
-                    {
-                        throw;
-                    }
```

Comment:
Why did you get rid of these rethrowings?

Reply:
I changed it to only catch `TimeoutException` here and throw all other exceptions.